doc/build-instructions/Build_Instructions_V15.md

```diff
@@ -0,0 +1,627 @@
+
+```
+bash configure \
+    --with-boot-jdk=<path_to_boot_JDK14> \
+    --with-freemarker-jar=/<my_home_dir>/freemarker.jar
+```
+
+:pencil: Modify the paths for freemarker and the macOS boot JDK that you installed in step 1. If `configure` is unable to detect Freetype, add the option `--with-freetype=<path to freetype>`, where `<path to freetype>` is typically `/usr/local/Cellar/freetype/2.9.1/`.
+
+:pencil: **Non-compressed references support:** If you require a heap size greater than 57GB, enable a noncompressedrefs build with the `--with-noncompressedrefs` option during this step.
+
+:pencil: **OpenSSL support:** If you want to build an OpenJDK that includes OpenSSL, you must specify `--with-openssl=path_to_library`, where `path_to_library` specifies the path to the prebuilt OpenSSL library that you obtained in **2. Get the source**. If you want to include the OpenSSL cryptographic library in the OpenJDK binary, you must also include `--enable-openssl-bundling`.
+
+### 4. build
+:apple:
+Now you're ready to build OpenJDK with OpenJ9:
+
+```
+make all
+```
+:warning: If you just type `make`, rather than `make all` your build will be incomplete, because the default `make` target is `exploded-image`.
+If you want to specify `make` instead of `make all`, you must add `--default-make-target=images` when you run the configure script.
+
+Two builds of OpenJDK with Eclipse OpenJ9 are built and stored in the following directories:
+
+- **build/macosx-x86_64-server-release/images/jdk**
+- **build/macosx-x86_64-server-release/images/jdk-bundle**
+
+    :pencil: For running applications such as Eclipse, use the **-bundle** version.
+
+    :pencil: If you want a binary for the runtime environment (jre), you must run `make legacy-jre-image`, which produces a jre build in the **build/macosx-x86_64-server-release/images/jre** directory.
+
+### 5. Test
+:apple:
+For a simple test, try running the `java -version` command. Change to the jdk directory:
+```
+cd build/macosx-x86_64-server-release/images/jdk
+```
+Run:
+```
+./bin/java -version
+```
+
+Here is some sample output:
+
+```
+openjdk version "15-internal" 2020-09-01
+OpenJDK Runtime Environment (build 15-internal+0-adhoc.jenkins.Build-JDK15-osxx86-64cmprssptrs)
+Eclipse OpenJ9 VM (build tye-e85051733, JRE 15 Mac OS X amd64-64-Bit Compressed References 20200901_27 (JIT enabled, AOT enabled)
+OpenJ9   - e85051733
+OMR      - dfbca14c
+JCL      - 2ef6b4c54d8 based on jdk-15+30)
+```
+
+:pencil: **OpenSSL support:** If you built an OpenJDK with OpenJ9 that includes OpenSSL support, the following acknowledgements apply in accordance with the license terms:
+
+- *This product includes software developed by the OpenSSL Project for use in the OpenSSL Toolkit. (http://www.openssl.org/).*
+- *This product includes cryptographic software written by Eric Young (eay@cryptsoft.com).*
+
+:ledger: *Congratulations!* :tada:
+
+----------------------------------
+
+## ARM
```

Comment:
While AArch64 is possible, I don't expect an ARM build is going to happen, this section should be removed.

Reply:
You removed AArch64, but I expected ARM to be removed.